tests/skipjob/access-policy-job/00-application.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: Application
```

Comment:
`Application` under the SkipJob tests? Is this to test that resources don't overlap?

Reply:
Its to test that the network policy egress rule is added. If no application is present in the namespace for the outbound rule, no rule is added